### PR TITLE
json compare with jq

### PR DESCRIPTION
**Pull request by nonsense**
This PR is adding a draft diff functionality with `jq` where we ignore the `_meta` field.

---

TODO:
- [ ] update `go get` to fetch `@next` for Lotus (or specs-actors) once the conformance PR is merged.
- [ ] schedule a CircleCI cron to periodically run `make compare-with-lotus-next`
- [ ] report jobs outcomes to Slack

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,8 @@ GENCOMMIT = `git rev-list -1 HEAD`
 gen:
 	find gen/suites -maxdepth 1 -mindepth 1 -type d -print0 | xargs -I '{}' -n1 -0 bash -c 'dir="$$(basename {})" && echo "=== $${dir} ===" && cd {} && go run -ldflags "-X github.com/filecoin-project/test-vectors/gen/builders.GenscriptCommit=${GENCOMMIT}" . -o "../../../corpus/$${dir}"'
 
-
+compare-with-lotus-next:
+	pushd gen && go get github.com/filecoin-project/specs-actors@master && go get github.com/filecoin-project/lotus@a0c0d9c98aae && popd && cp -R corpus corpus-current && make gen && mv corpus corpus-new && ./diff.sh
```
Comment (raulk) [inline Makefile:11]: Hm... how about a parametrisable target that takes two arguments?

```shell script
$ make compare-corpus lotus next
$ make compare-corpus specs-actors master
```

* Argument 1: a string that's interpolated with `github.com/filecoin-project/${arg}`
* Argument 2: a string that's passed as-is as the argument after the `@` in go get.

Comment (raulk) [reply Makefile:11]: Maybe make all this magic part of the diff script so that: (a) we can easily invoke it locally, (b) it cleans up after itself?

Comment (nonsense) [reply Makefile]: @raulk I liked that idea and I implemented it, but at the moment we can't upgrade specs-actors to master as our tool doesn't compile anymore, so I am not convinced we can just update lotus or specs-actors and expect for the test vectors to continue to be generated.

**Comment by raulk** [comment]
We ended up doing this in the `Generator` itself through different modes (`-u: update if needed, disregarding metadata`).

**Pull request closed without merging**